### PR TITLE
Use cljsjs/babel-standalone for js modules guide

### PR DESCRIPTION
**Pull request by Deraen**
https://github.com/cljsjs/packages/tree/master/babel-standalone includes the necessary JS file and a namespace with js-transforms implementation: https://github.com/cljsjs/packages/blob/master/babel-standalone/src/cljsjs/babel_standalone.clj

In future, tooling (figwheel, boot-cljs) might automatically require the namespace using the namespace part of preprocess dispatch key.

### CODE REVIEW

**Comment by Deraen** [comment]
As @thheller mentioned in Slack, it might be better to leave the guide mostly alone so the basics are covered. Cljsjs package could be mentioned after Babel Transforms part.

I'm leaving this here for a while for comments.

**Comment by arichiardi** [comment]
I would leave the first part of the description as well there for a while, it is such a new thing that it is good to see how it really works.

Then you can say: "but `(require ...)` does it for you".

Thanks this looks very useful overall.

**Comment by swannodette** [comment]
I actually really like this change but I agree we should probably have a new section which is something like `Custom JavaScript Transforms` which shows the long form.

File: content/guides/javascript-modules.adoc [hunk 1]
```diff
@@ -247,36 +247,17 @@ Change your project.clj to the following:
                  [org.clojure/clojurescript "1.9.473"]
```
Comment (Deraen) [inline content/guides/javascript-modules.adoc]: This needs to be updated before release.

**Comment by swannodette** [comment]
@Deraen is this ready?

**Comment by Deraen** [comment]
Just updated the Cljs version number, ready now.

**Pull request merged**